### PR TITLE
LSPD and ems offical rank from wiki

### PR DESCRIPTION
**Pull request by evanillaa**
**Describe Pull request**
First, make sure you've read and are following the contribution guidelines and style guide and your code reflects that.
Write up a clear and concise description of what your pull request adds or fixes and if it's an added feature explain why you think it should be included in the core.

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)
- Does your code fit the style guidelines? [yes/no]
- Does your PR fit the contribution guidelines? [yes/no]

### CODE REVIEW

**Review by Eralpozcan — CHANGES_REQUESTED** [review]
isboss should be, Chief of Police

**Comment by evanillaa** [comment]
@Eralpozcan done

**Review by Eralpozcan — CHANGES_REQUESTED** [review]
Please check "}," and space beetween, in police Readability is bad.

File: shared.lua [hunk 1]
```diff
@@ -603,55 +603,87 @@ QBShared.Jobs = {
...
+            },
+            ['11'] = {
+                name = 'Assistant Chief',
+                payment = 325
+            },
+            ['12'] = {
+		name = 'Chief of Police',
+		isboss = true,
+		payment = 350
+            },
```
Comment (Newtonzz) [inline shared.lua:661]: Spotted by Quarentin, necessary to change or the PR won't be accepted.

**Review by Newtonzz — CHANGES_REQUESTED** [review]
Please check your committed changes as they have minor errors that need fixing pointed out by Quarentin.

**Pull request closed without merging**